config/routes.rb

```diff
@@ -1,10 +1,9 @@
 Rails.application.routes.draw do
   resources :cards
-
-  # get 'home/index'
-  # get "home" => "home#index"
-  # get "/" => "home#index"
-  root  'home#index'
+  root  "home#index"
+  # match "/check",    to: "cards#check",    via: "patch"
+  # match "/cards/:id/check",    to: "cards#check",    via: "patch"
+  patch '/cards/:id/check' => 'cards#check', as: :check
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.